nomad/local/grapl-local-infra.nomad

```diff
@@ -6,6 +6,21 @@ variable "container_registry" {
 
 # The following variables are all-caps to clue in users that they're
```

Comment:
btw if you make changes to a nomad or packer file, you'll want to run `make format-hcl` otherwise you might fail our linter on CI (which it appears this change has). 

It's annoying, I know haha.

Reply:
Yep, I realized that after I pushed it up 🤦 . I've fixed that for the moment